aiohttp/web_runner.py

```diff
@@ -45,20 +45,18 @@ def _raise_graceful_exit() -> None:
 
 
 class BaseSite(ABC):
-    __slots__ = ("_runner", "_shutdown_timeout", "_ssl_context", "_backlog", "_server")
+    __slots__ = ("_runner", "_ssl_context", "_backlog", "_server")
 
     def __init__(
         self,
         runner: "BaseRunner",
         *,
-        shutdown_timeout: float = 60.0,
```

Comment:
For the 3.9 backport, I'll need to readd this for backwards-compatibility, with a deprecation.